user-guide/Standard_Apps/PA/Getting_Started_with_PA/Deploying_and_Configuring_the_PA_Framework.md

```diff
@@ -0,0 +1,114 @@
+
+## Installing the SRM framework
+
+Process Automation requires the creation of a [Booking Manager element](xref:SolSRM) named *Process Automation instances*, configured with the *Process Automation* virtual platform. To allow us to create this element, we first need to install the SRM framework on the DataMiner System.
+
+> [!NOTE]
+> In case SRM is already installed on your system, the procedure below is not required to proceed.
+
+To install the SRM framework:
+
+1. Ensure that a DataMiner version is installed that is compatible with the SRM framework, and the platform meets the hardware requirements.
+
+   - Make sure the DataMiner System uses an Elasticsearch database.
+
+   - Check the SRM release notes to see which minimum DataMiner version is required.
+
+   - Contact Skyline Communications to check if any additional components need to be installed in your system.
+
+1. Download the latest SRM package from [DataMiner Dojo](https://community.dataminer.services/downloads/).
+
+1. Double-click the SRM package, and install the package in the same manner as a DataMiner upgrade.
+
+   > [!TIP]
+   > See also: [Upgrading a DataMiner Agent](xref:Upgrading_a_DataMiner_Agent)
+
+   > [!NOTE]
+   > DataMiner will restart during the installation of the package.
+
+## Installing the Process Automation framework
+
+To install the PA framework:
+
+1. Ensure that a DataMiner version is installed that is compatible with the PA framework, and the platform meets the hardware requirements.
+
+   - Make sure the DataMiner System uses an Elasticsearch database.
+
+   - Check the Process Automation release notes to see which minimum DataMiner version is required.
+
+   - Contact Skyline Communications to check if any additional components need to be installed in your system.
+
+1. Download the latest PA package from [DataMiner Dojo](https://community.dataminer.services/downloads/).
+
+1. Double-click the PA package, and install the package in the same manner as a DataMiner upgrade.
+
+   > [!NOTE]
+   > DataMiner will restart during the installation of the package.
+
+   > [!TIP]
+   > See also: [Upgrading a DataMiner Agent](xref:Upgrading_a_DataMiner_Agent)
+
+## Configuring the PA framework
+
+To set up the initial configuration of the PA framework:
+
+1. Go to *Automation* module in DataMiner Cube and select the *SRM_Setup* Automation script.
+
+1. In the lower right corner, select *Execute*.
+
+1. In the *Booking Manager Element Info* box, specify the value `{}`.
+
+1. Click *Execute now*.
+
+![SRM Setup](~/user-guide/images/SRM_setup.png)
+
+> [!NOTE]
+> If the script execution fails, wait a few minutes and try again. If the problem persists, contact Skyline Communications.
+
+The script will configure the framework and create all relevant components to get started with the development of a process.
+
+## Upgrading the PA framework
```

Comment:
can we maybe move that section before 'Configure the PA Framework' ( thisis to avoid people run the script for a fresh installation, while they should first run the migration script )